CITATION.cff

```diff
@@ -0,0 +1,43 @@
+# YAML 1.2
+# More than one affiliation:
+# https://github.com/citation-file-format/citation-file-format/issues/268
+---
+title: EUCP Data Catalogue
+keywords:
+  - climate
+  - EUCP
+repository-code: "https://github.com/eucp-project/data-catalogue"
+license: "Apache-2.0"
+message: "If you use this software, please cite it using these metadata."
+cff-version: "1.2.0"
+authors:
+  -
+    family-names: Kalverla
+    given-names: Peter
+    orcid: "https://orcid.org/0000-0002-5025-7862"
+    affiliation: "Netherlands eScience Center"
+  -
+    family-names: Alidoost
```

Comment:
@SarahAlidoost just checking that you agree to being listed as contributor. I think you've mostly contributed feedback during demo's, right?

Reply:
Thanks for asking, you can remove me from the list as I did not have any active contributions in this repo. :)